src/test/ui/traits/trait-object-with-self-in-projection-output-bad.rs

```diff
@@ -0,0 +1,22 @@
+trait Base {
```

Comment:
Let's add a comment to this test:

---

Regression test for #56288. Checks that if a supertrait defines an associated type projection that references `Self`, then that associated type must still be explicitly specified in the `dyn Trait` variant, since we don't know what `Self` is anymore.